src/verify.c

```diff
@@ -704,7 +724,14 @@ int verify_main(int argc, char **argv)
 	times = init_timelist();
 
 	grabtime_start(&times, "Load and recurse Manifests");
-	official_manifest = load_mom(version, false);
+
+	/* If upstream URL is passed, user is likely trying to get back to the official stream,
+	 * so ignore any custom mixer related state */
+	if (strncmp(content_url, "https://download.clearlinux.org/update", 38) == 0) {
```

Comment:
Instead of hardcoding the upstream, which we are trying to get away from, why not detect that it is non-empty? if they specify a content_url you are probably right in assuming they want to get back to upstream, but we definitely shouldn't only make that assumption when it points to `https://specific.clear.url`.

Anyways, this ignores the perfectly valid `https://cdn.download.clearlinux.org/update`

Reply:
Probably right and assuming are not good enough IMO. Another option would be to compare it to the URLs listed in the /usr/share/defaults/swupd/* and compare those, then it wouldn't have to be maintained here.